marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.tsx

```diff
@@ -241,13 +245,7 @@ const MappingCard: React.FC<Props> = (props) => {
     const updateMappingWithNamespaces = async (mapDataLocal) => {
```

Comment:
@xnikhil08  @srinathgit  Does this support many namespaces in the source document? It looks like fetchSrcDocFromUri is parsing the value of "@xmlns"?? This really seems like something the backend should be doing. If this is just preserving QS functionality, I suppose it's fine for now. But can you create a ticket to rework this so that the backend does this properly, which include support namespaces at multiple levels of the document?

Reply:
yeah, based on preliminary analysis, it does currently support multiple namespaces (in multiple levels) in the source doc.